sklearn/metrics/tests/test_regression.py

```diff
@@ -56,6 +56,16 @@ def test_regression_metrics(n_samples=50):
                         np.sum(1 / y_true) / (4 * n))
 
 
+def test_mean_squared_error_multioutput_raw_value_squared():
```

Comment:
Please add a comment e.g.

`# non-regression test for https://github.com/scikit-learn/scikit-learn/pull/16323`